src/main/java/com/ixortalk/organization/api/rest/OrganizationRestController.java

```diff
@@ -131,7 +131,7 @@
                                 organization
                                         .getUsers()
                                         .stream()
-                                        .filter(user -> enhanceUserService.isAdmin(user))
+                                        .filter(User::isAdmin)
```

Comment:
@steinmaerivoet @wjans I think this change is breaking, unless the isAdmin column is kept in sync with auth0 before the release.
Are you aware of that?

Reply:
Yes, this is breaking indeed.  This PR has been merged and had a minor version bump upon realeasing (version `1.1.0`).  Minor version bumps in the IxorTalk platform indicate that changes are not backwards compatible.

Reply:
Another thing that has changed/improved here is that users will also be exposed as being an admin, even before they accepted their invitation.  However, both `status` and `isAdmin` are exposed, so it's up to clients to deal with this properly.